cpp/roaring.hh

```diff
@@ -112,6 +121,18 @@ public:
         return ans;
     }
 
+    /**
+     * Construct a bitmap from a list of uint32_t values.
+     * E.g., bitmapOf({1,2,3}).
+     */
+    static Roaring bitmapOf(std::initializer_list<uint32_t> l) {
```

Comment:
I have a slight concern with overloading `bitmapOf` (i.e. using the same name) in the context of people that like to use uniform initialization syntax (curly braces rather than parens).

Before this change, `bitmapOf({0})` would have been a valid call (to the `va_list` version), yielding a bitmap of size zero.
After this change, `bitmapOf({0})` now makes a bitmap of size 1, with its element 0 set.

I can't think of any other real-world examples that are likely to trip people up, but perhaps for the avoidance of confusion, the initializer_list version could be called something different like `bitmapOfList`.

Reply:
Fixed in a commit.